_playwright-tests/yarn.lock

```diff
@@ -879,6 +879,13 @@ consola@^3.2.3, consola@^3.4.0:
   resolved "https://registry.yarnpkg.com/consola/-/consola-3.4.0.tgz#4cfc9348fd85ed16a17940b3032765e31061ab88"
   integrity sha512-EiPU8G6dQG0GFHNR8ljnZFki/8a+cQwEQ+7wpxdChl02Q8HXlwEZWD5lqAF8vC2sEC3Tehr8hy7vErz88LHyUA==
 
+"content-sources-test-utils@file:./test-utils":
```

Comment:
I though this would also drop `pg` since you moved it, but I guess not?

Reply:
well the `pg` package was removed from the main package json, but it isn't removed because now it is being picked up as a dependency from the test utils 😄